server/controllers/users.js

```diff
@@ -400,6 +401,61 @@ module.exports = (app) => {
       .catch(next)
     },
 
+    /**
+     * Send an email with refreshed token
+     */
```

Comment:
"For the case when a user has submitted an expired token, we can automatically detect the email address and send a refreshed token"